src/components/MultiHopTrade/components/TradeInput/components/TradeQuotes/TradeQuote.tsx

```diff
@@ -236,7 +236,7 @@ export const TradeQuoteLoaded: React.FC<TradeQuoteLoadedProps> = ({
       <Flex justifyContent='space-between' alignItems='center'>
         <Flex gap={2} alignItems='center'>
           <SwapperIcon swapperName={quoteData.swapperName} />
-          <RawText>{quoteData.swapperName}</RawText>
+          <RawText>{quote?.steps[0].source ?? quoteData.swapperName}</RawText>
```

Comment:
Ditto here, currently fine with single-step Li.Fi quotes but as we implement actual multi-hop Li.Fi quotes, this will only give us the name of the first source

<img width="564" alt="image" src="https://github.com/shapeshift/web/assets/17035424/a4ffd2a0-80ac-4d76-b3e5-235c9f23df48">

Though writing this and double-checking against Jumper, this is the same way Jumper is displaying multi-hop quotes, so we may actually want to keep it that way in the future as well

<img width="429" alt="image" src="https://github.com/shapeshift/web/assets/17035424/6a574aa4-6918-426c-8b25-19d00a6ad87c">
<img width="423" alt="image" src="https://github.com/shapeshift/web/assets/17035424/1454bcb0-70ca-41b5-872d-b7886cc39170">